packages/CodingStandard/src/Fixer/Property/ArrayPropertyDefaultValueFixer.php

```diff
@@ -0,0 +1,103 @@
+<?php declare(strict_types=1);
+
+namespace Symplify\CodingStandard\Fixer\Property;
+
+use Nette\Utils\Strings;
+use PhpCsFixer\DocBlock\DocBlock;
+use PhpCsFixer\Fixer\DefinedFixerInterface;
+use PhpCsFixer\FixerDefinition\FixerDefinition;
+use PhpCsFixer\FixerDefinition\FixerDefinitionInterface;
+use PhpCsFixer\Tokenizer\CT;
+use PhpCsFixer\Tokenizer\Token;
+use PhpCsFixer\Tokenizer\Tokens;
+use SplFileInfo;
+
+final class ArrayPropertyDefaultValueFixer implements DefinedFixerInterface
+{
+    public function getDefinition(): FixerDefinitionInterface
+    {
+        return new FixerDefinition(
+            'Array property should have default value, to prevent undefined array issues.',
+            []
+        );
+    }
+
+    public function isCandidate(Tokens $tokens): bool
+    {
+        // analyze only properties with comments
+        return $tokens->isAllTokenKindsFound([T_DOC_COMMENT, T_VARIABLE]);
+    }
+
+    public function isRisky(): bool
+    {
+        return false;
+    }
+
+    public function fix(SplFileInfo $file, Tokens $tokens): void
+    {
+        foreach ($tokens as $index => $token) {
```

Comment:
Some fixers do this, typically fixers that add tokens at some point.
It tends to become an issue when adding tokens while there are more candidates behind it.
For utest we add multiple fix candidates for this after each other.
In a abstract way, lets say we replace `A` with `BC` in
`XA` => `XBC`; all good
`XAYA` => XBCYA ; because we stopped iterating at `Y`, expected was `XBCYBC`
do the reverse will not cause this issuel
`XAYA` => `XAYBC` (first fix), `XBCYBC` after second